package.json

```diff
@@ -4,7 +4,11 @@
   "description": "node.js_HW",
   "main": "server.js",
```

Comment:
Is it correct way to file?

Reply:
I have made corrections in the 'main': 'build/server.js'. Thank you!